dojo/db_migrations/0115_usercontactinfo_force_password_reset.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.13 on 2021-07-07 04:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0114_cyclonedx_vuln_uniqu'),
```

Comment:
@blakeaowens We have merged 2 PRs with db migrations, the last one is `0116_test_type_active`. I am afraid you have to rename this file and update the dependency.

Reply:
Yes, I see that now. Will take care of it!